### PR TITLE
Rectivate tooltips after table update

### LINKED ISSUE #125: Explanation for Block Trend
It would be nice if the UI had a summary of how the block trend is computed. The `i` tooltip trigger does not actually cause any tooltip to appear on hover.

image

### PR DESCRIPTION
**Pull request by enterprisey**
Fix #125

### CODE REVIEW

**Comment by gcochard** [comment]
Please [rebase your branch](https://stackoverflow.com/a/7244456) with this repo's latest master branch.

**Comment by enterprisey** [comment]
@gcochard :+1:

File: battleground-state-changes.html.tmpl [hunk 1]
```diff
@@ -320,7 +320,8 @@
                     if (latestContent.updates.states_updated.length) {
                         Array.from(document.getElementsByTagName("table")).forEach(table => {
                             let newTable = latestContent.dom.getElementById(table.id);
-                            table.parentNode.replaceChild(newTable, table);
+                            table.replaceChild(newTable, table);
```
Comment (gcochard) [inline battleground-state-changes.html.tmpl]: Looks like this line needs to be reverted.

**Comment by gcochard** [comment]
I can't merge this, but if the one line I commented on is fixed then it should be good to go.

File: battleground-state-changes.html.tmpl (hunk 1 above)
Comment (enterprisey) [reply battleground-state-changes.html.tmpl]: Sure does.

**Comment by titanous** [comment]
This was fixed in #183.

**Pull request closed without merging**